common/io/src/fasta_parser.cpp

```diff
@@ -20,14 +19,9 @@ namespace claragenomics
 namespace io
 {
 
-std::unique_ptr<FastaParser> create_hts_fasta_parser(const std::string& fasta_file)
+std::unique_ptr<FastaParser> create_kseq_fasta_parser(const std::string& fasta_file, int min_sequence_length)
 {
-    return std::make_unique<FastaParserHTS>(fasta_file);
-}
-
-std::unique_ptr<FastaParser> create_kseq_fasta_parser(const std::string& fasta_file)
-{
-    return std::make_unique<FastaParserKseqpp>(FastaParserKseqpp(fasta_file));
+    return std::make_unique<FastaParserKseqpp>(FastaParserKseqpp(fasta_file, min_sequence_length));
```

Comment:
Optional: No need to create a temporary `FastaParserKseqpp`, just call
`std::make_unique<FastaParserKseqpp>(fasta_file, min_sequence_length)`.

Reply:
done